extensions/functions/tensorFlowGeneral/resize.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 EMQ Technologies Co., Ltd.
```

Comment:
Try to merge this to the original resize function or remove from this PR. Let the resize function to have args of (x,y,z, channels, format)

Reply:
ack